pkg/healthcheck/healthcheck.go

```diff
@@ -136,6 +136,10 @@ const (
 	/// plugin is installed and ready
 	LinkerdCNIPluginChecks CategoryID = "linkerd-cni-plugin"
 
+	// LinkerdOpaquePortsDefinitionChecks adds checks to validate that the opaque-ports
+	// label has been defined both in the service and the corresponding deployment
```

Comment:
very nitpicky 😅 
```suggestion
	// label has been defined both in the service and the corresponding pods
```

The reason for this nitpicky comment is that the annotation should be put in the pod template spec of the deployment rather than the deployment itself. I've seen many people (myself included) put the annotation directly on the deployment and wondering why it doesn't work. I think it would be helpful for people that are not as familiar with how linkerd works to have this distinction reflected in the comment. Wdyt?

Reply:
You're totally right!